test/helper.sh

```diff
@@ -98,72 +102,196 @@ wait_for_managed_cert() {
     return 1
 }
 
-# Wait for the given ingress to completely delete its resources.
+# Get forwarding rule of the given ingress.
+# Arguments:
+#   Name of the ingress.
+#   Namespace of the ingress.
+#   The context where the resource resides.
+# Outputs:
+#   Name of http(s) forwarding rule for the ingress.
```

Comment:
... if both HTTP and HTTPS forwarding rules exist, this will return the HTTPS forwarding rule.